src/main/java/ca/corefacility/bioinformatics/irida/ria/web/ajax/projects/settings/ProjectSettingsAjaxController.java

```diff
@@ -17,37 +16,27 @@
 import ca.corefacility.bioinformatics.irida.ria.web.ajax.projects.settings.dto.Coverage;
 import ca.corefacility.bioinformatics.irida.ria.web.ajax.projects.settings.exceptions.UpdateException;
 import ca.corefacility.bioinformatics.irida.ria.web.services.UIPipelineService;
-import ca.corefacility.bioinformatics.irida.ria.web.services.UIProjectSettingsService;
-import ca.corefacility.bioinformatics.irida.security.permissions.project.ProjectOwnerPermission;
-import ca.corefacility.bioinformatics.irida.service.ProjectService;
-import ca.corefacility.bioinformatics.irida.service.user.UserService;
+import ca.corefacility.bioinformatics.irida.ria.web.services.UIProjectsService;
 
 /**
  * Controller to handle all asynchronous call from the project settings UI.
  */
 @RestController
 @RequestMapping("/ajax/projects/{projectId}/settings")
 public class ProjectSettingsAjaxController {
-	private final ProjectService projectService;
-	private final UserService userService;
-	private final ProjectOwnerPermission projectOwnerPermission;
 	private final UIPipelineService pipelineService;
-	private final UIProjectSettingsService settingsService;
+	private final UIProjectsService projectsService;
 
 	@Autowired
-	public ProjectSettingsAjaxController(ProjectService projectService, ProjectOwnerPermission projectOwnerPermission,
-			UserService userService, UIPipelineService pipelineService, UIProjectSettingsService settingsService) {
-		this.projectService = projectService;
-		this.projectOwnerPermission = projectOwnerPermission;
-		this.userService = userService;
+	public ProjectSettingsAjaxController( UIPipelineService pipelineService, UIProjectsService projectsService) {
 		this.pipelineService = pipelineService;
-		this.settingsService = settingsService;
+		this.projectsService = projectsService;
 	}
 
 	/**
 	 * Update the priority for analyses for a project.
 	 *
-	 * @param projectId identifier for a {@link Project}
```

Comment:
Shouldn't this remain as {@link Project}. Sorry I think I missed this before

Reply:
No, I try to avoid actually linking unless I am using the object in that class already